spikeinterface/extractors/neoextractors/openephys.py

```diff
@@ -96,32 +103,59 @@ class OpenEphysBinaryRecordingExtractor(NeoBaseRecordingExtractor):
     name = "openephys"
     has_default_locations = True
 
-    def __init__(self, folder_path, stream_id=None, stream_name=None, block_index=None, all_annotations=False):
-        neo_kwargs = self.map_to_neo_kwargs(folder_path)
-        NeoBaseRecordingExtractor.__init__(self, stream_id=stream_id, 
+    def __init__(self, folder_path, load_sync_channel=False, experiment_names=None,
+                 stream_id=None, stream_name=None, block_index=None, all_annotations=False):
+        neo_kwargs = self.map_to_neo_kwargs(folder_path, load_sync_channel, experiment_names)
```

Comment:
neo_kwargs = self.map_to_neo_kwargs(folder_path, load_sync_channel=load_sync_channel, experiment_names=experiment_names)

Reply:
I think that's ok